server/services/bookingService.test.ts

```diff
@@ -72,6 +80,26 @@ describe('BookingService', () => {
     })
   })
 
+  describe('getKeyworkers', () => {
+    it('should return the keyworker data needed', async () => {
+      const keyWorkers = referenceDataFactory.buildList(2)
```

Comment:
if we're to be consistent with the name of the method this should be `keyworkers`

Reply:
I've renamed the method, as the endpoint is `key-workers`